emission/net/ext_service/habitica/sync_habitica.py

```diff
@@ -0,0 +1,68 @@
+# Standard imports
+import json
+import requests
+import logging
+import arrow
+import attrdict as ad
+
+# Our imports
+import emission.core.get_database as edb
+import emission.net.ext_service.habitica.proxy as proxy
+import emission.analysis.result.metrics.simple_metrics as earmts
+import emission.analysis.result.metrics.time_grouping as earmt
+
+
+
+def create_habit(user_id, new_habit):
```

Comment:
This doesn't actually create the habit, it just retrieves it, right?
(you are calling `GetUserTasks`)

Where are the tasks created? Do you expect the user to create them? Are they created directly from the phone?